.travis/deploy-linux.bash

```diff
@@ -31,11 +31,11 @@ if [ "$DEPLOY_APPIMAGE" = "true" ]; then
 	# Package it up and send it off
 	./squashfs-root/usr/bin/appimagetool /rpcs3/build/appdir
 	ls
-	COMM_TAG="$(git describe --tags $(git rev-list --tags --max-count=1))"
+	COMM_TAG="$(cat rpcs3/rpcs3_version.cpp | grep 'version{.*}' | awk -F[{,] '{printf \"%d.%d.%d\", $2, $3, $4}')"
```

Comment:
this can bi simplified to:
```
COMM_TAG="$(grep 'version{.*}' rpcs3/rpcs3_version.cpp | awk -F[{,] '{printf \"%d.%d.%d\", $2, $3, $4}')"
```

Reply:
Sure, I just do `cat | grep` out of habit

Reply:
Well at that point may as well do the whole thing with awk.

Reply:
Too confusing, it's better to leave it with grep + awk, I made it grep every between version{ } on purpose so it's more resilient to changes